specs/extension_image.md

```diff
@@ -35,8 +35,109 @@ confext images extend `/etc`. They must contain a `/etc/extension-release.d/exte
 to identify them.
 
 ## Image Content
-Extension Images should be additive, and not override content present in the base image or other DDIs,
-but this will not be enforced.
+Extension Images should be additive, and not override content present in the base image or other DDIs.
+However, there currently is no safe and efficient way to detect collisions and to enforce content uniqueness
+across the stack of images.
+In a future version of this specification options for enforcing uniqueness may be provided.
+
+## Base Directory Immutability / Mutability
+The contents of system and configuration extension images are immutable and cannot be changed after
```

Comment:
I think this should say "by default" or otherwise give a hint that it's not always true.

Reply:
No, the _extension contents_ are always immutable. The whole sentence is misleading though since that section discusses base directory mutability. I'll remove it.